tests/ignite/contrib/handlers/test_tensorboard_logger.py

```diff
@@ -1,5 +1,6 @@
 import math
 import os
+import unittest
```

Comment:
We are using pytest so let's stick to their formalism without creating TestCase classes and just directly defining test functions.